### DISCUSSION TITLE
Yourls API „Beauty enhancement“

### DISCUSSION THREAD

**Discussion by XHyperDEVX** [YOURLS/YOURLS#3552] [General]
Hello,

I am using the Yourls Api of my instance and have the problem that when there are no links in the database the Api gives the following: (Stats)

`{"stats":{"total_links":0,"total_clicks":null},"statusCode":200,"message":"success"}`

Can I change this to that:

`{"stats":{"total_links":0,"total_clicks":0},"statusCode":200,"message":"success"}`

Thanx

**Comment by ozh** [comment]
Easy answer : no, and that shouldn't be much of a problem with your app using the API, simply convert the value to an integer ?

Less easy answer : this is probably doable with a plugin -- see https://yourls.org/docs/guide/extend/possible-with-a-plugin/
